### PR TITLE
PACA-733: RDA API data dictionary plugin.

### PR DESCRIPTION
**Pull request by brianburton**
**JIRA Ticket:**
[PACA-733](https://jira.cms.gov/browse/PACA-733)

**User Story or Bug Summary:**

Add tooling to automate the creation of coding system descriptions in the BB2 static website.

### What Does This PR Do?

Follows the conventions established for generating pages for CCW codebook variables to generate similar pages for FISS and MCS claims from the RDA API data dictionary.

- Adds a generator plugin to process records from the data dictionary and produce one page per record.
- Adds a layout file to render the pages.
- Adds a CSV file to `_data` containing the RDA API data dictionary records to render.
- Adds commented out sample configuration to `_config.yaml`.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check these things, in particular:

* Does this follow conventions.
* Is there more to be done to properly integrate with the site.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence: N/A.
    * Does this PR add any new software dependencies? **Yes** or **No**.
    * Does this PR modify or invalidate any of our security controls? **Yes** or **No**.
    * Does this PR store or transmit data that was not stored or transmitted before? **Yes** or **No**.
* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] I have named this PR and its branch such that they'll be automatically be linked to the (most) relevant Jira issue, per: <https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html>.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

**Review by ericbuckley — COMMENTED** [review]
@brianburton looks great so far, would you mind also amending [this section of the README](https://github.com/CMSgov/bluebutton-site-static#codebook) to explain the differences between the plugins reading the _codebooks dir and this new one that reads out of _data.

**Review by ericbuckley — APPROVED** [review]

**Review by KyRobbins — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -88,3 +88,10 @@ For information about deploying to the static site, please referernce the follow
 
 ## Codebook
 The [`_codebook` directory](_codebooks/) and the [corresponding Jekyll plugin](_plugins/api_codings.rb) are used to create the variables resources, such as [this one](https://bluebutton.cms.gov/resources/variables/clm_non_utlztn_days_cnt/). Although these *URLs* show little to no usage from analytics, they are used as *URIs* and are **integral to the [Beneficiary FHIR Data (BFD) endpoint responses](https://github.com/CMSgov/beneficiary-fhir-data/tree/b1ca6c4b630f6b0e5aa7e812addabf72bc334fe3/apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses)**. For now (Dec 2021), changing the URIs would require completely retiring all of the BFD v1 and v2 APIs and releasing a new v3 with different URIs. Any additions or changes to these URIs should involve the BFD team.
```
Comment (gmaciolek) [inline README.md:90]: Suggest updating this sentence to:
The [`_codebook` directory](_codebooks/) and the [corresponding Jekyll plugin](_plugins/api_codings.rb) are used to create the variables resources.. Example: [Claim Medicare Non Utilization Days Count](https://bluebutton.cms.gov/resources/variables/clm_non_utlztn_days_cnt/).

Comment (brianburton) [reply README.md]: Thanks!  I have made that change and a similar one to the RDA section at the end.

Comment (gmaciolek) [reply README.md]: Looks good to me!

**Review by gmaciolek — COMMENTED** [review]
Approved. Thanks for checking with me!

**Review by gmaciolek — APPROVED** [review]

**Pull request merged**